lib/orogen/spec/project.rb

```diff
@@ -136,6 +144,11 @@ def default_task_superclass
             # Task contexts are represented as instances of TaskContext. See
             # the documentation of that class for more details.
 	    def task_context(name, options = Hash.new, &block)
+                if namespace_disabled?(name.split("::")[0..-2].join("::"))
+                    OroGen::Gen::RTT_CPP.warn ("Task #{name} is disabled, because the namespace " + name.split("::")[0..-2].join("::") + " is disabled")
```

Comment:
This should *DEFINITELY* not be a warning. This will be a perfectly common and normal situtation !